connectors/opa/lib/opa_reader_test.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2020 IBM Corp.
+// SPDX-License-Identifier: Apache-2.0
+
+package opaconnbl
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	tu "github.com/ibm/the-mesh-for-data/connectors/opa/testutil"
+)
+
+// Tests  GetPoliciesDecisions in opa-connector.go
+// In the test the purpose is set as "marketing". For this purpose in different scenarios connector is mocked and different outputs are obtained.
+// In this test the results are  manually synchronised, result of customOpaResponse function should be translated into
+// GetExpectedOpaDecisions. Tested here is the functionality of translating opa_result
+// format into enforcement decisions format
+
+func TestMainOpaConnector(t *testing.T) {
+
+	timeOutSecs, catalogConnectorURL, opaServerUrl := tu.GetEnvironment()
+	applicationContext := tu.GetApplicationContext("marketing")
+
+	srv := NewOpaReader(opaServerUrl)
+	catalogReader := NewCatalogReader(catalogConnectorURL, timeOutSecs)
+	policiesDecisions, _ := srv.GetOPADecisions(applicationContext, catalogReader)
```

Comment:
You should check that the error is nil:
```
policiesDecisions, err := srv.GetOPADecisions(applicationContext, catalogReader)
assert.NilError(t, err)
```

Reply:
Fixed by using the assert function from gotest.tools/assert. Thanks.